python/lsst/obs/base/makeRawVisitInfo.py

```diff
@@ -62,23 +63,33 @@ class MakeRawVisitInfo(object):
     However, the methods that transform units are less forgiving; they assume
     the user provides proper data types, since type errors in arguments to those
     are almost certainly due to coding mistakes.
+
+    Parameters
+    ----------
+    `lsst.log.Log`
+        Logger to use for messages.
+        (None to use ``Log.getLogger("MakeRawVisitInfo")``).
     """
 
     def __init__(self, log=None):
-        """Construct a MakeRawVisitInfo
-        """
         if log is None:
             log = Log.getLogger("MakeRawVisitInfo")
         self.log = log
 
     def __call__(self, md, exposureId):
-        """Construct a VisitInfo and strip associated data from the metadata
-
-        @param[in,out] md  metadata, as an lsst.daf.base.PropertyList or PropertySet;
-            items that are used are stripped from the metadata
-            (except TIMESYS, because it may apply to more than one other keyword).
-        @param[in] exposureId  exposure ID
-
+        """Construct a VisitInfo and strip associated data from the metadata.
+
+        Parameters
+        ----------
+        md : `lsst.daf.base.PropertyList` or `PropertySet`
```

Comment:
Does sphinx find `PropertySet` without it being `~lsst.daf.base.PropertySet`?

Reply:
Probably not? We'll need some kind of pydoc linter to find all the `py:obj` contexts that don't get mapped, I suspect. It's hard to manually catch all such cases when converting from doxygen.